lib/rialto/etl/cli/composite_etl.rb

```diff
@@ -106,18 +107,27 @@ def file_prefix
           raise NotImplementedError
         end
 
+        # rubocop:disable Metrics/MethodLength
         def extract(row, profile_id, force)
           extract_file = File.join(input_directory, "#{file_prefix}-#{profile_id}.ndj")
           if File.exist?(extract_file) && !force
             say "file #{extract_file} already exists, skipping. use -f to force overwrite"
             return extract_file
           end
-          results = perform_extract(row)
+          # Don't write file if an exception occurs
+          begin
+            results = perform_extract(row)
+          rescue StandardError
+            say "Skipping #{extract_file} because an error occurred while extracting"
```

Comment:
What would you think about surfacing the exception class and message here too? Useful, or not so much?

Reply:
Done.